.ci/jenkins/selenium/run_tests.sh

```diff
@@ -16,6 +16,8 @@ fi
 mkdir -p "$GALAXY_TEST_ERRORS_DIRECTORY"
 mkdir -p "$GALAXY_TEST_SCREENSHOTS_DIRECTORY"
 
+docker run -v `pwd`:`pwd`:rw -w `pwd` -u $UID node /bin/bash -c 'make client-production'
```

Comment:
The jenkins job has `make client-production-maps` - I am not sure that it makes a difference here.

Reply:
Yeah, I swapped that thinking we might as well save the little bit of time.  If the selenium test output actually resolves through the sourcemaps, and it's useful, I can definitely swap it back.

Reply:
I believe the sourcemaps do produce better outputs in the tests - such as for the captured console logs.

Reply:
Hrmm, I didn't think they actually resolved through the test output, but I can swap it.  Really, for the best test output, we should just do 'make client', but then it's not testing the production optimizations.

Reply:
It is possible they weren't - I didn't have evidence just intuition and vague recollections.